src/QMCDrivers/CloneManager.cpp

```diff
@@ -131,21 +123,15 @@ void CloneManager::makeClones(MCWalkerConfiguration& w,
   wClones.resize(NumThreads);
   PsiPoolClones.resize(NumThreads);
   HPoolClones.resize(NumThreads);
-  wClones[0]       = &w;
-  PsiPoolClones[0] = psipool;
-  HPoolClones[0]   = hampool;
```

Comment:
objects for thread 0 are owned elsewhere and passed in while objects for other threads are cloned and therefore owning pointers.

I tried making just this change to the develop branch and get the same test failures.